### DISCUSSION TITLE
How to send mail in text/html rather than text/plain

### DISCUSSION THREAD

**Discussion by ghost** [argoproj-labs/argocd-notifications#291] [Q&A]
When I send a mail with ArgoCD notifications I'd like to be able to send a "nice templated mail" using HTML content. I can see that currently mails sent by ArgoCD are sent in `text/plain` format.

![image](https://user-images.githubusercontent.com/12391965/119241385-3a063900-bb56-11eb-85c1-edc2085b110c.png)

Is there a simple workaround to send mails in `text/html`?

**Comment by ryota-sakamoto** [comment]
Hi @TPX01 
I created issue to enhancement email, because it is not supported yet.
https://github.com/argoproj/notifications-engine/issues/29
